home/default.nix

```diff
@@ -11,6 +11,8 @@
   # Recommended Nix settings
   nix = {
     registry.nixpkgs.flake = flake.inputs.nixpkgs; # https://yusef.napora.org/blog/pinning-nixpkgs-flake/
+    # Garbage collect automatically every week
+    gc.automatic = true;
```

Comment:
What if I want to change the frequency? Can you add that code and comment it out?